ui/app/src/App.tsx

```diff
@@ -11,53 +11,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Box, useTheme } from '@mui/material';
-import { ErrorAlert, ChartsThemeProvider, generateChartsTheme } from '@perses-dev/components';
-import { PluginRegistry, PluginBoundary } from '@perses-dev/plugin-system';
-import ViewDashboard from './views/ViewDashboard';
-import { DataSourceRegistry } from './context/DataSourceRegistry';
-import Header from './components/Header';
-import Footer from './components/Footer';
-import { useBundledPlugins } from './model/bundled-plugins';
+import { Routes, Route } from 'react-router-dom';
+import HomeDashboard from './views/HomeDashboard';
+import Docs from './views/Docs';
 
 function App() {
-  const { getInstalledPlugins, importPluginModule } = useBundledPlugins();
-
-  const muiTheme = useTheme();
-
-  // app specific echarts option overrides, empty since perses uses default
-  // https://apache.github.io/echarts-handbook/en/concepts/style/#theme
-  const echartsThemeOverrides = {};
-  const chartsTheme = generateChartsTheme('perses', muiTheme, echartsThemeOverrides);
-
   return (
-    <Box
-      sx={{
-        display: 'flex',
-        flexDirection: 'column',
-        height: '100vh',
-      }}
-    >
-      <Header />
-      <Box
-        component="main"
-        sx={{
-          flexGrow: 1,
-          overflow: 'hidden',
-        }}
-      >
-        <ChartsThemeProvider themeName="perses" chartsTheme={chartsTheme}>
-          <PluginRegistry getInstalledPlugins={getInstalledPlugins} importPluginModule={importPluginModule}>
-            <PluginBoundary loadingFallback="Loading..." ErrorFallbackComponent={ErrorAlert}>
-              <DataSourceRegistry>
-                <ViewDashboard />
-              </DataSourceRegistry>
-            </PluginBoundary>
-          </PluginRegistry>
-        </ChartsThemeProvider>
-      </Box>
-      <Footer />
-    </Box>
+    <Routes>
+      <Route path="/" element={<HomeDashboard />} />
```

Comment:
Ideally the routing should be something like `/projects/<project_name>/dashboards/<dashboard_name>`.

Like that when sharing URL, the FE will be able to know what is the dashboard to load

Reply:
Maybe we can discuss this again at the next Perses meeting, this would need to happen when we connect the BE and FE right? I really just wanted react-router in this PR since [useSearchParams](https://reactrouter.com/en/6.4.0/hooks/use-search-params) made things easy

Reply:
yeah that's why I said "ideally", but definitely that would have to happen when we will connect BE and FE. 
We can talk about it Monday no issue.

It can be done later for sure and move on following the changes you wanted to perform